angular.json

```diff
@@ -33,11 +33,11 @@
       "prefix": "app",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-devkit/build-angular:application",
           "options": {
             "outputPath": "dist/svg-icon",
             "index": "src/index.html",
-            "main": "src/main.ts",
+            "browser": "src/main.ts",
             "polyfills": "src/polyfills.ts",
```

Comment:
It should be an array for application builder

Reply:
@ectuser - Looks like it needs to be an array. After building the distribution, here's the error:

```
npm start

> svg-icon@17.0.0 start
> ng serve --configuration production

Node.js version v21.1.0 detected.
Odd numbered Node.js versions will not enter LTS status and should not be used for production. For more information, please see https://nodejs.org/en/about/previous-releases/.
Option "browserTarget" is deprecated: Use 'buildTarget' instead.
Error: Schema validation failed with the following errors:
  Data path "/polyfills" must be array.
```

Reply:
Yes, you're right, added in the https://github.com/czeckd/angular-svg-icon/pull/150/commits/03dd59dc9097b60312cba51313872314113ffbc9

Also renamed browserTarget property to buildTarget.